src/Project/Project.router.ts

```diff
@@ -228,8 +235,8 @@ export class ProjectRouter extends Router {
 
     addInmutableCacheControlHeader(res)
     return res.redirect(
-      `${getBucketURL()}/${project.getFileKey(filename)}`,
-      301
+      301,
+      `${getBucketURL()}/${project.getFileKey(filename)}`
```

Comment:
were these params in the wrong order all this time?

Reply:
It accepts both, but the one we had before was "deprecated"

```
    redirect(status: number, url: string): void;
    redirect(url: string, status: number): void;
```